api/column_family.cc

```diff
@@ -926,14 +926,14 @@ void set_column_family(http_context& ctx, routes& r, sharded<db::system_keyspace
     cf::enable_auto_compaction.set(r, [&ctx](std::unique_ptr<http::request> req) {
         apilog.info("column_family/enable_auto_compaction: name={}", req->get_path_param("name"));
         auto [ks, cf] = parse_fully_qualified_cf_name(req->get_path_param("name"));
-        validate_table(ctx, ks, cf);
+        auto ti = table_info{ .name = cf, .id = get_uuid(ks, cf, ctx.db.local()) };
```

Comment:
I don't understand what I'm seeing here :-( You are assigning ti and then not using it? What's the point?

Also, the new code is longer, uglier and has more duplication than the old code, so I still don't understand what's the point. The commit messages keeps saying that it will "allow making for_table_on_all_shards() smaller" but I have no idea how this change allows that.

Reply:
Oh, I see, it is used in a later patch....